src/schema/objects/metadata.yaml

```
@@ -140,6 +140,17 @@ AnatomicalLandmarkCoordinates__mri:
       type: number
     minItems: 3
     maxItems: 3
+ScreenAOIDefinition:
+  name: ScreenAOIDefinition
+  display_name: Screen Area of Interest Definition
+  description: |
+    Define, what shape are the Screen AOIs and what coordinates are used.
```

```suggestion
    A description of the shape of the Screen AOIs and what coordinates are used.
```

Maybe something like this? I'm not married to the wording though.